packages/search/lib/commands/AGGREGATE.ts

```diff
@@ -213,6 +214,10 @@ export function pushAggregatehOptions(
         args.push('DIALECT', options.DIALECT.toString());
     }
 
+    if (options?.TIMEOUT) {
```

Comment:
Is `TIMEOUT` `0` an option? I think that `options?.TIMEOUT !== undefined` is better..

Reply:
made the change
I think the same thing would be applicable for SEARCH as well
made changes there too